### PR TITLE
Fix token inline hook instructions

### PR DESCRIPTION
**Pull request by johngronberg-okta**
## Description:
The instructions for setting up token inline hooks are incorrect. The PR fixes the steps

### Resolves:

* [OKTA-219822](https://oktainc.atlassian.net/browse/OKTA-219822)

### CODE REVIEW

**Comment by imvenkat** [comment]
Looks good to me.

File: packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md [hunk 1]
```diff
@@ -304,13 +304,15 @@ To activate the inline hook, you first need to register your external service en
 
 You then need to associate the registered inline hook with a Custom Authorization Server Policy Rule by completing the following steps in Admin Console:
 
-1. Go to **Security > API Authorization Servers**.
+1. Go to **API > Authorization Servers**.
 
 1. Select the Custom Authorization Server to use this inline hook with.
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md]: Unless there will only be 1 custom authorization server, perhaps word it like:

> Select a Custom Authorization Server from the list.

File: packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md [hunk 2]
```diff
@@ -304,13 +304,15 @@ To activate the inline hook, you first need to register your external service en
 
 You then need to associate the registered inline hook with a Custom Authorization Server Policy Rule by completing the following steps in Admin Console:
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md]: drop `in Admin Console`

File: packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md [hunk 3]
```diff
@@ -304,13 +304,15 @@ To activate the inline hook, you first need to register your external service en
...
 
-1. Go to **Security > API Authorization Servers**.
+1. Go to **API > Authorization Servers**.
 
 1. Select the Custom Authorization Server to use this inline hook with.
 
-1. One of the rules defined in the Custom Authorization server needs to be used to trigger invocation of the inline hook. Click the pencil icon for that rule to open it for editing.
+1. Select the Access Policies tab, and select the policy you would like to use for the hook.
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md]: How about

> Select the Access Policies tab and and select a policy to use with the hook. In most cases, just pick the Default Policy.

File: packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md [hunk 4]
```diff
@@ -304,13 +304,15 @@ To activate the inline hook, you first need to register your external service en
...
 
-1. In the **Advanced Settings** section, click the **Assertion Inline Hook** dropdown menu. Any inline hooks you have registered will be listed. Select the one to use.
+1. One of the policy rules defined in the policy needs to be used to trigger invocation of the inline hook. Click the pencil icon for that rule to open it for editing.
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md]: How about

> One of the policy's rules needs to trigger the inline hook. Click the pencil for a rule to edit it. If you only have one rule, edit the Default Policy Rule.

Comment (nbarbettini) [inline packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md]: How about

> One of the policy's rules needs to trigger the inline hook. Click the pencil for a rule to edit it. If you only have one rule, edit the Default Policy Rule.

**Pull request closed without merging**